packages/swap/src/config/env.ts

```diff
@@ -45,6 +48,15 @@ export default z
     USE_JIT_QUOTING: optionalBoolean,
     QUOTE_APPROXIMATION_THRESHOLD: optionalNumber(0.1),
     STEALTH_MODE: optionalBoolean.default('true'),
+    DISABLED_INTERNAL_ASSETS: optionalString('').transform((string) =>
+      string.split(',').map((asset) => {
```

Comment:
we can use `forEach` here I think

Reply:
think we cannot use foreach because we need to return the array (this is inside the transform)